apps/meteor/tests/end-to-end/api/livechat/19-business-hours.ts

```diff
@@ -43,6 +48,38 @@ describe('[CE] LIVECHAT - business hours', function () {
 			expect(response.body.businessHour.workHours[0].finish).to.be.an('object');
 			expect(response.body.businessHour.workHours[0].open).to.be.a('boolean');
 			expect(response.body.businessHour.timezone).to.be.an('object').that.has.property('name').that.is.an('string');
+
+			defaultBhId = response.body.businessHour;
+		});
+		it('should not allow a user to be available if BH are closed', async () => {
+			await saveBusinessHour({
+				...defaultBhId,
+				workHours: [
+					{
+						day: 'Monday',
+						open: true,
+						start: '00:00',
+						finish: '00:01',
+					},
+				],
+			});
+
+			// @ts-expect-error - afsdf
```

Comment:
Why don't we type the response from `makeAgentAvailable` so we don't need to override the error here?